libraries/chain/transaction_context.cpp

```diff
@@ -259,12 +262,14 @@ namespace bacc = boost::accumulators;
              }
          }
       }
-      validate_ram_usage.reserve( bill_to_accounts.size() );
+
```

Comment:
Performance question. But it's better do `validate_ram_usage.reserve()` in this place. `bill_to_accounts` doesn't contains info about account, which will be use provider bandwidth. But this accounts can have ram changes.